docs/UserGuide.md

```diff
@@ -138,16 +138,21 @@ Examples:
 
 ### Filtering out flashcards  : `filter`
 
-Filters the specified flashcard based on category input(s) by user.
+Filters the specified flashcard based on the field input(s) by the user. 
```

Comment:
Hi ZY, maybe you could use "Filters for specific flashcards" instead of Filters the specified flashcard". I think this would be clearer to the readers.